server_src/lib/serverCommands.js

```diff
@@ -1,9 +1,14 @@
 const fs = require("fs");
+const path = require("path");
 const util = require("util");
 const exec = util.promisify(require("child_process").exec);
 const Logger = require("../lib/logger.js");
 const logger = new Logger("OctoFarm-Scripts");
```

Comment:
logger name does not correspond file name, that intended?

Reply:
It's me segregating it, this file needs splitting up I think. Server commands != The scripts feature. The server commands need to be in the OctoFarm-Server.log file and Scripts could either got in the API / their own. 

I don't generally group by file name for logging I split by system area so when an issue arrises I can get a single log file specific to the area of the issue, this doesn't really follow that since including the system commands so thanks for pointing that out.